src/forms/NewStudyForm.js

```diff
@@ -1,351 +1,453 @@
-import React, {useState} from 'react';
+import React, {useState, Fragment} from 'react';
 import PropTypes from 'prop-types';
 import {
   Form,
   Segment,
   Container,
-  Label,
   Button,
   Message,
-  Grid,
   Confirm,
   List,
   Dropdown,
+  Step,
 } from 'semantic-ui-react';
 import {Formik} from 'formik';
 import {workflowOptions} from '../common/enums';
+import {PopupFormField} from '../components/PopupFormField';
 
-const NewStudyForm = ({submitValue, apiErrors}) => {
-  const [confirmOpen, setConfirmOpen] = useState(false);
-  const [workflowType, setSelection] = useState([
-    'bwa_mem',
-    'gatk_haplotypecaller',
-  ]);
+const InfoStep = ({
+  formikProps,
+  setActiveStep,
+  setFocused,
+  focused,
+  setSelection,
+  workflowType,
+}) => {
+  const {values, errors, touched, handleChange, handleBlur} = formikProps;
+  return (
+    <>
+      <Message info>
+        Please provide the study's full name and a shortened version that may be
+        used for display purposes.
+      </Message>
+      <PopupFormField
```

Comment:
Since we now have more vertical space let's make all the field descriptions like below. If it's information that the users needs let's not hide it from them or make them work for it
![image](https://user-images.githubusercontent.com/1334131/65266089-5845a700-dae0-11e9-8ca6-c9fcea8c26b4.png)